content/en/docs/guides/server-graceful-stop.md

```diff
@@ -0,0 +1,86 @@
+---
+title: GracefulStop
+description: >-
+  Explains how to gracefully shut down a gRPC server.
+---
+
+### Overview
+
+gRPC servers often need to shut down gracefully, ensuring that in-flight RPCs
+are completed within a reasonable timeframe and new RPCs are no longer
+accepted. `GracefulStop()` facilitates this process, allowing the server to
```

Comment:
I know you are using "GracefulStop()" here as a placeholder for the language specific functions, but I still think it might still lead readers to look for a function with this exact name. How about just referring it to as the "graceful stop function"? It's more wordy but can avoid a potential confusion.

Reply:
Changed to "Graceful Shutdown Function"